modules/ingest-common/src/internalClusterTest/java/org/elasticsearch/ingest/common/ManyNestedPipelinesIT.java

```diff
@@ -40,6 +41,9 @@
  */
 public class ManyNestedPipelinesIT extends ESIntegTestCase {
     private final int manyPipelinesCount = randomIntBetween(2, 50);
+    private final int tooManyPipelinesCount = 102;
```

Comment:
This can be set to 101 if we decide to merge #105427.

Reply:
Sounds good, let's get that one merged in and then update this PR.